src/test/java/de/tum/in/www1/artemis/staticcodeanalysis/IntegrationTest.java

```diff
@@ -0,0 +1,108 @@
+package de.tum.in.www1.artemis.staticcodeanalysis;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.catchThrowableOfType;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.stream.Collectors;
+
+import org.junit.jupiter.api.Test;
+import org.xml.sax.SAXParseException;
+
+import de.tum.in.www1.artemis.service.connectors.localci.scaparser.ReportParser;
+import de.tum.in.www1.artemis.service.connectors.localci.scaparser.exception.ParserException;
+import de.tum.in.www1.artemis.service.connectors.localci.scaparser.utils.XmlUtils;
+
+/**
+ * Tests each parser with an example file
+ */
+class IntegrationTest {
```

Comment:
I would rename this test

Reply:
I renamed it to `StaticCodeAnalysisIntegrationTest`, is that better?